packages/kolibri-core-for-export/package.json

```diff
@@ -24,7 +24,7 @@
     "js-cookie": "^3.0.5",
     "knuth-shuffle-seeded": "^1.0.6",
     "kolibri-constants": "0.2.0",
-    "kolibri-design-system": "https://github.com/learningequality/kolibri-design-system#v2.0.0-beta1",
```

Comment:
I don't know why beta was here instead of `0ed2f274b1bc3808218a4d3f526c181b96b32c6d`. I assume it might be caused by a release -> develop merge conflict. According to the `yarn.lock` it seems that `0ed2f274b1bc3808218a4d3f526c181b96b32c6d` was the actually installed version so should be okay.